tests/components/Layout.test.tsx

```diff
@@ -97,6 +106,47 @@ describe('<Layout />', () => {
       })
     })
 
+    describe('mise à jour des chats', () => {
+      it("notifie quand c'est un nouveau message d'un jeune", async () => {
+        // Given
+        const unJeuneChatNonLu = unJeuneChat({
+          ...jeunes[0],
+          lastMessageSentBy: 'jeune',
+          chatId: `chat-${jeunes[0].id}`,
+          lastMessageContent:
+            'Ceci est tellement nouveau, donne moi de la notif',
+        })
+
+        // When
+        await act(async () => {
+          updateChatCall(unJeuneChatNonLu)
+        })
+
+        // Then
+        await waitFor(() => {
+          expect(mockAudio).toHaveBeenCalled()
+        })
+      })
+      it("ne notifie pas quand c'est une tierce modification du chat", async () => {
+        // Given
+        const unJeuneChatDejaVu = unJeuneChat({
```

Comment:
```suggestion
        const unJeuneChatDejaLu = unJeuneChat({
```
en miroir a unJeuneChatNonLu?

Reply:
tout a fait